docs/sp-migration/generate-manifests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+set -e
+
+version_aws=v0.37.0
+version_azure=v0.34.0
+version_gcp=v0.34.0
+
+rm -f "sp-manual.yaml" && touch "sp-manual.yaml"
+rm -f "sp-family-manual.yaml" && touch "sp-family-manual.yaml"
+kubectl get managed --no-headers -o jsonpath='{range .items[*]}{.apiVersion}{"\n"}{end}' | grep -E '(aws|gcp|azure).upbound.io' | sort | uniq | while read -r line
+do
+  service=$(echo "${line}" | cut -d. -f1)
+  provider=$(echo "${line}" | cut -d. -f2)
+  if [[ ${provider} == "upbound" ]]; then
```

Comment:
@ezgidemirel, could you make sure you took the recent changes done with https://github.com/upbound/upjet/pull/204

Reply:
thanks for the catch @turkenh, updated the script